Application/EdFi.Ods.Sandbox/Repositories/IClientAppRepo.cs

```diff
@@ -1,4 +1,4 @@
-﻿// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
i have seen this twice now, I am wondering was the file not UTF-8?

Reply:
There are a lot of files (particularly in Admin) that are checked in as UTF-8-BOM instead of UTF-8.  Removing the BOM usually leads to these sort of diffs, since GitHub doesn't have a good way to render that being removed.